resources/js/components/Listing/DeletesListingRow.js

```diff
@@ -0,0 +1,41 @@
+export default {
+  data() {
+    return {
+      deletingRow: false,
+    };
+  },
+
+  computed: {
```

Comment:
We probably don't need this `computed` code if it's just empty.

Reply:
Done!